index.html

```diff
@@ -2039,6 +2039,45 @@ <h3>Protecting Application Developers</h3>
         </p>
       </section>
 
+      <section>
+        <h3>Conventions for Naming Cryptography Suites</h3>
+        <p>
+Section <a href="#versioning-cryptography-suites"></a> emphasized the 
+importance of providing relatively easy to understand information concerning the
+timeliness of particular cryptographic suite. While section 
+<a href="#protecting-application-developers"></a> further emphasized minimizing 
+the number of options to be specified. Indeed section 
+<a href="#cryptographic-suites"></a> lists requirements for crytographic suites
+which include detailed specification of algorithm, transformation, hashing and 
+serialization. Hence the name of the cryptographic suite does not need to 
+include all this detail which implies the <em>parametric versioning</em> mentioned
+in section <a href="#versioning-cryptography-suites"></a> is not necessary or
+desirable.
+        </p>
+        <p>
+The recommended naming convention convention for cryptographic suites is a 
+string composed of a signature algorithm identifier, separated by a hyphen from 
+an option identifier (if the cryptosuite supports more than incompatible 
+implementation option), followed by a hyphen and then designation of the year 
+that the suite was approximately proposed.
+        </p>
+        <p>
+For example the [[DI-EDDSA]] is based on EdDSA digital signatures and supports 
+two incompatible options based on canonicalization approaches, and was proposed 
+roughly in the year 2022 would have two different cryptosuite names: 
```

Comment:
```suggestion
For example, the [[DI-EDDSA]] is based on EdDSA digital signatures, supports 
two incompatible options based on canonicalization approaches, and was proposed 
in roughly the year 2022, so it would have two different cryptosuite names: 
```